PySDM/builder.py

```diff
@@ -45,6 +45,12 @@ def add_dynamic(self, dynamic):
         assert key not in self.particulator.dynamics
         self.particulator.dynamics[key] = dynamic
 
+    def remove_dynamic(self, dynamic):
```

Comment:
let's add a simple unit test that would depict example usage (we already have `test_builder.py`)